src/tshwctl.c

```diff
@@ -34,24 +34,45 @@ void do_info(void)
 		printf("CPU_OPTS=0x%X\n", eval_cmd("cpu_opts"));
 		printf("IO_OPTS=0x%X\n", eval_cmd("io_opts"));
 		printf("IO_MODEL=0x%X\n", eval_cmd("io_model"));
+
```

Comment:
I'd argue to remove this empty line since the indenting provides differentiation.